CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Fall back to version 2 project config if version 3 fails. ([#1314](https://github.com/getsentry/relay/pull/1314))
 - Reduce number of metrics extracted for release health. ([#1316](https://github.com/getsentry/relay/pull/1316))
 - Indicate with thread is the main thread in thread metadata for profiles. ([#1320](https://github.com/getsentry/relay/pull/1320))
+- Revert fallback to v2 project config, v3 is reliable now. ([#1325](https://github.com/getsentry/relay/pull/1325))
```

Comment:
nit: Not sure what our policy is here, but when we revert a change within the same unreleased version, I would just remove both changelog items.

Reply:
i would agree with that actually

Reply:
I don't know, I removed them now.

Personally I wouldn't even keep internal changelog entries at all, so I just followed what was there.